### ISSUE TITLE
Toggle is triggered from previous Page

### ISSUE THREAD

**Issue by antoine92190** [driftyco/ionic#11148]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[ ] **2.x**
[x] **3.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
- I am on Page1 and I have an ion-list button to go to Page2 using "navCtrl.push(Page2)".
- On Page2, I have a toggle button.
- In Page1, when I click on the ion-list button, at the same position as the toggle is in Page2 (the right edge), it triggers the toggle in Page2.

**Expected behavior:**

Click propagation should be stopped when navigating from a Page to another.

**Steps to reproduce:**

**Related code:**

```
insert any relevant code here
```

**Other information:**

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
insert the output from ionic info here
```
Cordova CLI: 6.4.0 
Ionic Framework Version: 3.0.1
Ionic CLI Version: 2.2.1
Ionic App Lib Version: 2.2.0
Ionic App Scripts Version: 1.3.1
ios-deploy version: 1.9.0 
ios-sim version: 5.0.13 
OS: macOS Sierra
Node Version: v6.10.0
Xcode version: Xcode 8.3 Build version 8E162

**Comment by jgw96** [comment]
Hello, thanks for using Ionic! Are you seeing this on iOS, Android or both? Also, could you share a repo we could use to reproduce this issue?

**Comment by antoine92190** [comment]
I can experience this issue on iOS, haven't tried on Android.
You can reproduce it by using the following repo:

- git clone https://github.com/antoine92190/ionic-toggle.git
- npm install
- ionic platform add ios
=> run on Xcode simulator

Then, on the Home Page, click on the right arrow of the "Go to Page1" button.
You should see that the Page1 toggle is triggered without doing anything inside of Page1.

**Comment by antoine92190** [comment]
I cannot reproduce the issue anymore, was probably fixed on ionic 3.1.1
